main.py

```diff
@@ -22,17 +21,18 @@
 from src.client import client_env
 
 client = client_env()
-query="what is petofy"
+query="what is pet microchip"
```

Comment:
@nivesh1000 Make sure to pass these thru arguments

Reply:
resolved.

Reply:
now query can be passed through command line.